pyproject.toml

```diff
@@ -31,16 +30,3 @@ markers = [
     "flaky: flaky tests",
     "network: tests requiring a network connection",
 ]
-
-[tool.tox]
-legacy_tox_ini="""
-[tox]
-isolated_build = true
-extras = parallel
-minversion = 3.18.0
-envlist = py37, py38, py39
-
-[testenv]
-changedir = tests
-deps = pytest
-"""
```

Comment:
Since you're not using poetry, you could move (or port) the remaining of this file to setup.cfg

Reply:
I like having the tooling configuration separate from the build system.

Reply:
So in this case maybe move flake8 & mypy config out of setup.cfg?